integration/testdata/ca_cert_apps/no_package_manager/app.py

```diff
@@ -0,0 +1,6 @@
+import http.server, ssl
+
+server_address = ('0.0.0.0', 8080)
+httpd = http.server.HTTPServer(server_address, http.server.SimpleHTTPRequestHandler)
+httpd.socket = ssl.wrap_socket(httpd.socket, server_side=True, certfile='cert.pem', keyfile='key.pem', ssl_version=ssl.PROTOCOL_TLS)
+httpd.serve_forever()
```

Comment:
I do not think this app verifies the client against the CA certificate loaded by the buildpack.
Here's what I do
```
cd AccraZed/python/integration/testdata/ca_cert_apps
pack build caimg -p no_package_manager -b <your-cnb>
docker run --init -it -p 8080:8080 --env PORT=8080 --env SERVICE_BINDING_ROOT="/bindings" -v "$(pwd)/bindings:/bindings/ca-certificates" caimg
```

Now I expect (1) the following to fail with a certificate error
```
curl -k https://localhost:8080
```
and (2) the following to succeed
```
curl --cert client_certs/cert.pem --key client_certs/key.pem --cacert client_certs/ca.pem https://localhost:8080
```
But, both of them succeeds. This means the client auth is not checked.

Looking at the python api, One way might be to set [`cert_reqs=ssl.CERT_REQUIRED`](https://docs.python.org/3/library/ssl.html#ssl.wrap_socket) and `ca_certs` to the path to the system trust store. 
[`ssl.get_default_verify_paths()`](https://docs.python.org/3/library/ssl.html#ssl.get_default_verify_paths) might also be able to provide the system store path. If there's no straight-forward way, I'm also okay if you use `SSL_CERT_DIR` (see [ca-cert README](https://github.com/paketo-buildpacks/ca-certificates)) directly.

The same issue exists for the flask apps. You can take a look at ca-certificate test app in [nodejs](https://github.com/paketo-buildpacks/nodejs/tree/v0.6.2/integration/testdata/ca_cert_apps/node_server) or [go](https://github.com/paketo-buildpacks/go/tree/v0.10.0/integration/testdata/ca_certificate_apps/build) to compare the expected behavior

Reply:
I took a swing at this and looks like [SSLContext.load_default_certs()](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_default_certs) seems to work.
```py
# ssl
context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
context.verify_mode = ssl.CERT_REQUIRED
context.load_default_certs()
context.load_cert_chain(certfile='cert.pem', keyfile="key.pem")
# wrap http
server_address = ('0.0.0.0', 8080)
httpd = http.server.HTTPServer(server_address, http.server.SimpleHTTPRequestHandler)
httpd.socket = context.wrap_socket(httpd.socket, server_side=True)
httpd.serve_forever()
```